etc/kayobe/kolla.yml

```diff
@@ -369,7 +369,7 @@ kolla_build_customizations: "{{ kolla_build_customizations_common | combine(koll
 
 # Dict mapping Kolla Dockerfile ARG names to their values.
 kolla_build_args:
-  node_exporter_version: "1.5.0" # kolla has 1.4.0
+  node_exporter_version: "1.5.0"  # kolla has 1.4.0
```

Comment:
How could this terrible mistake have slipped through?

Reply:
Really catastrophic I know. Disaster averted :relieved: